src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1870,7 +1885,7 @@ private void SetPeriodicEvaluation(WorkspaceModel ws)
                 CustomNodeManager,
                 this.LinterManager);
 
-            workspace.FileName = filePath;
+            workspace.FileName = String.IsNullOrEmpty(filePath) ? "" : filePath;
```

Comment:
what affect does this have? Isn't workspace.FileName going to end up being `null` `""` or `filepath` in all cases?

Reply:
I just wanted to avoid having workspace.FileName set to null and set it to empty string instead so that I can mimic the exact FIleName setting of a new Home unsaved dyn file.